apache2/mod_security2.c

```diff
@@ -95,6 +95,7 @@ TreeRoot DSOLOCAL *conn_write_state_suspicious_list = 0;
 #ifdef WAF_JSON_LOGGING_ENABLE
 char DSOLOCAL *msc_waf_resourceId = "";
 char DSOLOCAL *msc_waf_instanceId = "";
+char DSOLOCAL *msc_waf_lock_perm = "root";
```

Comment:
Should this be set to DEFAULT_USER?

Reply:
thought about it, since they have both DEFAULT_USER and DEFAULT_GROUP and i don't want to add two cmds for us, since for us, the group and user always the same, and for our usage, it's ok to set the perm to root as default,  since nginx master have to run as root to open port 80. so i think it's fine